ansible_collections/arista/avd/plugins/plugin_utils/cv_client/client/change_control.py

```diff
@@ -0,0 +1,171 @@
+    """
+
+    workspace_api_version: Literal["v1"] = "v1"
+
+    async def get_change_control(
+        self: CVClient,
+        change_control_id: str,
+        time: datetime | None = None,
+        timeout: float = 10.0,
+    ) -> ChangeControl:
+        """
+        Get Change Control using arista.changecontrol.v1.ChangeControlService.GetOne API
+
+        Parameters:
+            change_control_id: Unique identifier of the Change Control.
+            time: Timestamp from which the information is fetched. `now()` if not set.
+            timeout: Timeout in seconds.
+
+        Returns:
+            ChangeControl object matching the change_control_id
+        """
+        request = ChangeControlRequest(
+            key=ChangeControlKey(id=change_control_id),
+            time=time,
+        )
+        client = ChangeControlServiceStub(self._channel)
+
+        try:
+            response = await client.get_one(request, metadata=self._metadata, timeout=timeout)
+            return response.value
+
+        except Exception as e:
+            raise get_cv_client_exception(e, f"Change Control ID '{change_control_id}'") or e
+
+    async def set_change_control(
+        self: CVClient,
+        change_control_id: str,
+        name: str | None = None,
+        description: str | None = None,
+        timeout: float = 10.0,
+    ) -> ChangeControlConfigSetResponse:
+        """
+        Set Change Control details using arista.changecontrol.v1.ChangeControlConfigService.Set API
+
+        Parameters:
+            change_control_id: Unique identifier of the Change Control.
+            name: Change Control Name.
+            description: Change Control description.
+            TODO: Add CC template
+            timeout: Timeout in seconds.
+
+        Returns:
+            ChangeControlConfig object after being set including any server-generated values.
+        """
+        request = ChangeControlConfigSetRequest(
+            value=ChangeControlConfig(
+                key=ChangeControlKey(id=change_control_id),
+                change=ChangeConfig(name=name, notes=description),
+            )
+        )
+        client = ChangeControlConfigServiceStub(self._channel)
+
+        try:
+            response = await client.set(request, metadata=self._metadata, timeout=timeout)
+            return response.value
+
+        except Exception as e:
+            raise get_cv_client_exception(e, f"Change Control ID '{change_control_id}'") or e
+
+    async def approve_change_control(
+        self: CVClient,
+        change_control_id: str,
+        timestamp: _DateTime,
+        description: str | None = None,
+        timeout: float = 10.0,
+    ) -> ApproveConfig:
+        """
+        Get Change Control using arista.changecontrol.v1.ChangeControlService.GetOne API
+
+        Parameters:
+            change_control_id: Unique identifier the Change Control.
+            timestamp: Timestamp for the change control information to be approved. \
```

Comment:
```suggestion
            timestamp: Timestamp for the change control information to be approved.
```

Reply:
This escape is needed for intellisense to pick up a multiline description.